xds/internal/clusterspecifier/rls/rls_test.go

```diff
@@ -107,56 +108,64 @@ func (s) TestParseClusterSpecifierConfig(t *testing.T) {
 }
 
 // This will error because the required match field is set in grpc key builder.
-var rlsClusterSpecifierConfigError = testutils.MarshalAny(&rlspb.RouteLookupClusterSpecifier{
-	RouteLookupConfig: &rlspb.RouteLookupConfig{
-		GrpcKeybuilders: []*rlspb.GrpcKeyBuilder{
-			{
-				Names: []*rlspb.GrpcKeyBuilder_Name{
-					{
-						Service: "service",
-						Method:  "method",
+func rlsClusterSpecifierConfigError(t *testing.T) *anypb.Any {
```

Comment:
Given that vars `rlsClusterSpecifierConfigError` and `rlsClusterSpecifierConfigWithoutTransformations` are only ever referenced from the test table above, please inline them instead of making a function for them.